src/XRP/serializer.ts

```diff
@@ -71,6 +71,12 @@ interface PaymentTransactionJSONAddition extends PaymentJSON {
   TransactionType: 'Payment'
 }
 
+interface PathJSON {
```

Comment:
Should this be called `PathElementJSON` perhaps?  since a `Path` is a separate thing?

Reply:
Done